paper/implementation.tex

```diff
@@ -75,11 +75,11 @@ \subsubsection{Toolbar functionalities}
     For the File Editor, this opens a dialog to open a file from the local file system.
     For the Schema Editor, this comes with four sub-options:
     \begin{itemize}
-        \item JSON Schema Store: Choose a schema from the JSON Schema Store, which provides a list of available schemas.
+        \item JSON Schema Store: Choose a schema from the JSON Schema Store, the biggest online collection of JSON schemas.
 
-        \item From Our Examples: Access four example schemas provided within the tool.
+        \item From Our Examples: Access one of the example schemas inbuilt into the tool.
```

Comment:
The option don't align with the text in the gui
![image](https://github.com/PaulBredl/config-assistant/assets/34026908/be7e5bcd-c69d-46f3-979a-006df27f8def)

Reply:
@PaulBredl better now?

Reply:
I actually ment the text isn't the same, and I thought the item elements would represent the actual menu text. But I guess the meaning is clear